core/modules/simpletest/tests/menu.test

```diff
@@ -49,7 +49,7 @@ class MenuWebTestCase extends BackdropWebTestCase {
     $pass = ($pass && empty($parts));
 
     $this->assertTrue($pass, format_string('Breadcrumb %parts found on @path.', array(
-      '%parts' => implode(' » ', $trail),
+      '%parts' => implode(' ', $trail),
```

Comment:
> I'm not able to dsm (or the like) to see what this function is getting from the page, or what it's building from the data being passed to it for testing against the page. Not sure how to debug this :cry:

To see what the test is seeing, enable the Testing module on the modules page, then run the test from Configuration -> Development -> Testing. When a test does an HTTP request to load a page, it creates a "Verbose output" link in the results table. Click on this link and you can see the full HTML page that the test bot requested.

Sometimes this can really help, as there may be a situation where the breadcrumb is getting checked in multiple themes, so so removing the `»` might fix one test but break another.